### ISSUE TITLE
Show Drain info after adopted

### ISSUE THREAD

**Issue by beyondtheteal** [citizenlabsgr/adopt-a-drain#150]
## Problems
* Information shown in the "Unadopted view" can never be viewed again. 
* Your drains are completely impersonal, even compared to others. 

### 1 - Unadopted view
While selecting a non-adopted drain, you can where it discharges to and the municipality.
<img width="248" alt="screen shot 2019-03-02 at 1 42 49 pm" src="https://user-images.githubusercontent.com/39837332/53686097-1d6d5480-3cf1-11e9-963e-a4c58f0e7138.png">

### 2 - Adopted-by-you view
After it is adopted, you can only see the "thank you" screen again
<img width="1001" alt="screen shot 2019-03-02 at 1 42 28 pm" src="https://user-images.githubusercontent.com/39837332/53686132-645b4a00-3cf1-11e9-9044-b0f0fe0c31ea.png">

### 3 - Adopted-by-other view
When you select someone else's drain, you can see their custom drain name and business name.
<img width="252" alt="screen shot 2019-03-02 at 1 43 36 pm" src="https://user-images.githubusercontent.com/39837332/53686172-ca47d180-3cf1-11e9-85a1-c6a6cd5e0e41.png">

### 4 - Sign-in prompt view
<img width="213" alt="screen shot 2019-03-02 at 5 20 11 pm" src="https://user-images.githubusercontent.com/39837332/53688273-9085c380-3d0f-11e9-83f5-9f9123a87d9f.png">

## Proposed solution 
Create a new view that is a combination of "Unadopted" and "Adopted by other" views. 
It should contain the following information. 

### Required
* Your custom drain name, ~and an ability to edit it.~
* Your name and your business name, as others would view it. 
* (existing) Ability to Abandon, with warning. 
* (existing) Ability to share
* Where the drain discharges to and the municipality

### Optional based on discussion
* ~Date you adopted the drain~

(Crossed out features should be moved to their own issue)

**Comment by cara-lgrow** [comment]
If we can display the date the drain was adopted, that would be good. I am assuming that this data is at least being kept in the back end of the program somewhere? That way we can automatically send emails at the 1 year mark (or other pre-determined amt of time), asking people if they want to renew their adoption.

@beyondtheteal  Can we change the 'Ability to share' after the drain is adopted? Facebook needs to be included, twitter can be deleted (all autofill is incorrect). Do you think that the email should stay? If so, the auto populated email text needs to change (basin->watershed) and the link to the app will eventually have to change.

**Comment by beyondtheteal** [comment]
@cara-lgrow, 
I've moved the Date requirement to issue #176 since we're not already storing it. 

I don't think I'm qualified to comment on if the email should be used. Rest assured that the URL is coming from whatever environment we're in - so if the email is sent from Staging, it will link to Staging. 

I was able to remove the Twitter icon and add the Facebook icon, but I need the copy for the Facebook link. Thanks!
